projects/storefrontlib/src/lib/cms-lib/category-navigation/category-navigation.component.scss

```diff
@@ -4,29 +4,13 @@
 @import '~bootstrap/scss/nav';
 @import '~bootstrap/scss/dropdown';
 
-//this hack here for now, until we decide how we go about keyboard navigation
-// .dropdown:hover>.dropdown-menu,
-// .dropdown:focus>.dropdown-menu {
-//   display: block;
-// }
-
-//navigation color variables
-$cx-navigation-list-background-color: 'inverse' !default;
-
-//navigation variables
-$cx-navigation-list-trbl-padding: 1rem 0 !default;
-
-.cx-navigation {
-  &__list {
-    display: flex;
-    justify-content: space-between;
-    padding: $cx-navigation-list-trbl-padding;
-    @include media-breakpoint-down(md) {
-      flex-direction: column;
-      @include var-color(
-        'background-color',
-        $cx-navigation-list-background-color
-      );
-    }
+:host {
+  display: flex;
+  justify-content: space-between;
+  padding: var(--cx-padding, 1rem 0);
```

Comment:
We talked about this but when resizing I thought the element would wrap but the navigation elements breakup which look quite bad. Maybe you could just add `flex-wrap:wrap;`.

Reply:
**Currently** ![screen shot 2019-01-15 at 2 43 42 pm](https://user-images.githubusercontent.com/10605724/51205719-8aa95f80-18d4-11e9-9227-1197e63a7535.png)

**I want**  ![screen shot 2019-01-15 at 2 47 26 pm](https://user-images.githubusercontent.com/10605724/51205760-a3197a00-18d4-11e9-9b4e-8296989e4cc4.png)